frontend/lib/map/map/attribution_dialog.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import 'attribution_dialog_item.dart';
+
+class AttributionDialog extends StatelessWidget {
+  const AttributionDialog({Key key})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SimpleDialog(
+      title: Text('Kartendaten'),
+      children: [
+        AttributionDialogItem(
+          icon: Icons.copyright,
+          color: Colors.blue,
+          text: 'OpenStreetMap',
+          onPressed: () {
+            launch("https://www.openstreetmap.org/about/");
+          },
+        ),
```

Comment:
According to https://www.openstreetmap.org/copyright, the text should say "© OpenStreetMap-Mitwirkende", and the link should go to https://www.openstreetmap.org/copyright